src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -1466,6 +1467,7 @@ internal virtual bool TryApplyChanges(Solution newSolution, IProgress<CodeAnalys
 
             foreach (var projectChanges in projectChangesList)
             {
+                progressTracker.Report(CodeAnalysisProgress.Description(string.Format(WorkspacesResources.Applying_changes_to_0, projectChanges.NewProject.Name)));
```

Comment:
now, when we're doing the *actual* application of hte changes to the workspace, you now see progress for that as well.